protocol/protocol.proto

```diff
@@ -221,6 +222,7 @@ message MemberPayload {
         Nft nft = 7;
         Pin pin = 8;
         Unpin unpin = 9;
+        bool mlsEnabled = 10;
```

Comment:
variable should be snake cased in the protobuf def (because python)
I would have gone with `optional string encryption_algorithm` to give more flexibility in the future.